src/allreduce_robust.cc

```diff
@@ -79,6 +83,68 @@ void AllreduceRobust::SetParam(const char *name, const char *val) {
     num_local_replica = atoi(val);
   }
 }
+
+int AllreduceRobust::SetCache(const std::string &key, const void *buf,
```

Comment:
Are `SetCache()` and `GetCache()` only used for bootstrap stage? If so, let's rename them to `SetBootstrapCache()` and `GetBootstrapCache()` respectively.

Reply:
yes, fixed.